server/controllers/lighthouse.js

```diff
@@ -72,6 +72,15 @@ function getResults (input) {
       },
     },
   };
+  const funcScoreChannelWeight = { // 100 LBC adds 1 point to the score
+    'function_score': {
+      'script_score': {
+        'script': {
+          'source': `${effectiveFactor} * doc['certificate_amount'].value`,
```

Comment:
are you sure this works? I'd expect it to be
```suggestion
          'source': `${effectiveFactor * doc['certificate_amount'].value}`,
```

(and same above)